tests/test_validate.py

```diff
@@ -609,7 +609,7 @@ def test_run_path_artifact_brew(tmpdir, caplog):
 
     img_desc = image_descriptor.copy()
     img_desc['artifacts'] = [{'name': 'aaa',
-                              'md5': 'd41d8cd98f00b204e9800998ecf8427e',
```

Comment:
Note this is a legit md5 in brew, so the fetch actually works, I changed it to something that doesn't exist (for now), but it might be better to pick something like: deadbeefdeadbeefdeadbeefdeadbeefdea not that that wouldn't occur either I suppose, but the intent is a bit clearer.